### PR TITLE
CDS-73133-limitation for service overrides on multi env set up

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/get-started/services-and-environments-overview.md [hunk 1]
```diff
@@ -548,6 +548,9 @@ Service overrides are different from **Environment Configuration** in the follow
 * Environment **Configuration**: applies to every service that is used with the environment.
 * Environment **Service Overrides**: applies to specific services you select. Whenever that service is used with that environment, the **Service Override** is applied.
 
+:::note
+Runtime inputs are not supported if you are trying to override services in a multi environment set up.
```
Comment (tchaurasiya) [inline docs/continuous-delivery/get-started/services-and-environments-overview.md:552]: Runtime inputs are not supported in both multi service or multi environment case

**Review by tchaurasiya — CHANGES_REQUESTED** [review]

File: docs/continuous-delivery/get-started/services-and-environments-overview.md (hunk 1 above)
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/services-and-environments-overview.md:552]:
```suggestion
Runtime inputs are not supported if you are trying to override services in a multi service and multi environment set up.
```

File: docs/continuous-delivery/get-started/services-and-environments-overview.md [hunk 2]
```diff
@@ -548,6 +548,9 @@ Service overrides are different from **Environment Configuration** in the follow
 * Environment **Configuration**: applies to every service that is used with the environment.
 * Environment **Service Overrides**: applies to specific services you select. Whenever that service is used with that environment, the **Service Override** is applied.
 
+:::note
+Runtime inputs are not supported if you are trying to override services in a multi service and multi environment set up.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/get-started/services-and-environments-overview.md:552]:
```suggestion
Runtime inputs are not supported if you are trying to override services in a multi-service and multi-environment set up.
```

File: docs/continuous-delivery/x-platform-cd-features/environments/service-overrides.md [hunk 3]
```diff
@@ -10,6 +10,9 @@ To enable the same service to use different environment settings, DevOps teams c
 
 This topic explains what service settings can be overridden by environments.
 
+## Limitations
+
+* Runtime inputs are not supported if you are trying to override services in a multi service and multi environment set up.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/environments/service-overrides.md:15]:
```suggestion
* Runtime inputs are not supported if you are trying to override services in multi-service and multi-environment set ups.
```

**Review by tchaurasiya — APPROVED** [review]

**Pull request merged**